lib/github.js

```diff
@@ -157,12 +157,23 @@ class GitHubApi {
     )
   }
 
+  lockIssue(issue, lockReason) {
+    return this.dryRunGuardedRequest(
+      `/issues/${issue.number}/lock`,
+      'put',
+      // eslint-disable-next-line camelcase
+      request => request.type('json').send({lockReason: lockReason || 'resolved'})
```

Comment:
is this supposed to be `lock_reason`?
```suggestion
      request => request.type('json').send({lock_reason: lockReason || 'resolved'})
```